conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -468,40 +460,31 @@ public void testBinaryServerSideFailureAfterFewBytesSent() {
     @Test
     @Timeout(20)
     public void testBinaryServerSideFailureAfterFewBytesReceived() {
-        byte[] data = new byte[1024];
+        int chunkSize = 1024;
+        int expectedChunks = 1024; // 1 MB
+        int chunksToSend = 1024 * 64; // 64 MB
```

Comment:
Still needs attention. This is systemically slower than the previous implementation and if I were to match this with 1GB instead of 64MB it would blow the 20 second timeout budget. I was not able to figure out why on my first try.